cmd/ocsp-updater/main_test.go

```diff
@@ -114,7 +114,7 @@ func TestGenerateAndStoreOCSPResponse(t *testing.T) {
 	err = updater.storeResponse(meta)
 	test.AssertNotError(t, err, "Couldn't store certificate status")
 
-	secondMeta, err := updater.generateRevokedResponse(ctx, status)
+	secondMeta, _, err := updater.generateRevokedResponse(ctx, status)
```

Comment:
Please add a test case that checks the new return value of this method (URLs to purge).